src/main/java/echoServer/http/HTTPRequest.java

```diff
@@ -0,0 +1,15 @@
+package echoServer.HTTP;
+
+public class HTTPRequest {
+
+    public String requestType;
+    public String route;
+    public String httpType;
+
+    public HTTPRequest(String requestType, String route, String httpType) {
+        this.requestType = requestType;
+        this.route = route;
+        this.httpType = httpType;
```

Comment:
I would encourage you to really pay attention to what the accepted nomenclature is.

The three parts of an HTTP request line are: the method, the URI or resource, and the HTTP version number.